frontend/src/pages/modelRegistrySettings/CreateMRSecureDBSection.tsx

```diff
@@ -25,22 +27,25 @@ interface CreateMRSecureDBSectionProps {
   secureDBInfo: SecureDBInfo;
   modelRegistryNamespace: string;
   nameDesc: { name: string };
-  existingCertKeys: string[];
-  existingCertConfigMaps: string[];
-  existingCertSecrets: string[];
+  existingCertConfigMaps: ConfigSecretItem[];
+  existingCertSecrets: ConfigSecretItem[];
   setSecureDBInfo: (info: SecureDBInfo) => void;
 }
 
 export const CreateMRSecureDBSection: React.FC<CreateMRSecureDBSectionProps> = ({
   secureDBInfo,
   modelRegistryNamespace,
   nameDesc,
-  existingCertKeys,
   existingCertConfigMaps,
   existingCertSecrets,
   setSecureDBInfo,
 }) => {
-  const [searchValue, setSearchValue] = useState('');
+  const [configSecretName, setConfigSecretName] = useState('');
+  const [key, setKey] = useState('');
+  const [existingCertKeys, setExistingCertKeys] = useState<string[]>([]);
```

Comment:
If we have `secureDBInfo` to tell us which secret or configmap is selected, we can derive the list of keys in it at render time and then we don't need this extra state for `existingCertKeys` I think?

Reply:
Yes, that's right. Removed this state altogether after adding `resourceType` to `SecureDBInfo`